### DISCUSSION TITLE
洪蘭：人腦拚電腦　贏在人文素養

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#193] [General]
https://www.cw.com.tw/article/5080299
"如果電腦無法取代的是以人文為核心的職業，我們就得加強學生的人文素養，來保障他們的工作。"文章中的這段話讓我印象深刻

**Comment by cobb110202008** [comment]
情感及素養的確是人獨有的，人工智慧要擁有這些人獨有的特質仍遙遙無期。

**Comment by wh20502** [comment]
我覺得人文素養也有可能被AI取代，就音樂的部分，我們平時聽的都是人創作的，但是已經有AI能夠訓練出模型，來完成一些音樂作品，搞不好以後會聽AI做的歌曲。AI完成莫札特音樂：https://news.ltn.com.tw/news/world/breakingnews/3008575

**Comment by Shiba010** [comment]
現在連ai都可以進行藝術創作了，未來真的還很難說。
